examples/all-clusters-app/all-clusters-common/gen/IMClusterCommandHandler.cpp

```diff
@@ -5318,7 +5318,7 @@ void DispatchServerCommand(app::Command * apCommandObj, CommandId aCommandId, En
         {
         case ZCL_ADD_OP_CERT_COMMAND_ID: {
             expectArgumentCount = 4;
-            chip::ByteSpan OperationalCert;
+            chip::ByteSpan NOCArray;
```

Comment:
This change will get overwritten when someone regenerates the code from the ZAP files.
Also, suggest to keep renaming as a separate PR. It'll help review the core changes in the current PR.

If we want to keep the rename in this PR, we should update the XML and ZAP files, and regenerate the code.

Reply:
Agree about renaming this one later

Reply:
We're updating the operational-credentials-cluster.xml and ZAP files in this PR already to address the name change.